Rubberduck.Parsing/VBA/SynchronousReferenceRemover.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Rubberduck.VBEditor;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace Rubberduck.Parsing.VBA
+{
+    public class SynchronousReferenceRemover : ReferenceRemoverBase 
+    {
+        public SynchronousReferenceRemover(
+            RubberduckParserState state,
+            IModuleToModuleReferenceManager moduleToModuleReferenceManager) 
+        :base(
+            state, 
+            moduleToModuleReferenceManager)
+        { }
+
+        public override void RemoveReferencesTo(ICollection<QualifiedModuleName> modules, CancellationToken token)
+        {
+            if (!modules.Any())
+            {
+                return;
+            }
```

Comment:
doesn't do anything useful. if there are no modules, the foreach just doesn't execute